src/editor/index.js

```diff
@@ -113,6 +115,11 @@ class PopupSidebar extends Component {
 					min={ 0 }
 					max={ 100 }
 				/>
+				<CheckboxControl
```

Comment:
Can we use `FormToggle` instead?

Reply:
Good idea. Done in https://github.com/Automattic/newspack-popups/pull/51/commits/2a014a29fe73d4b1cb45405382812ecebcc104c7